modular_skyrat/modules/synths/code/bodyparts/power_cord.dm

```diff
@@ -93,6 +93,10 @@
 		user.nutrition += power_use / SYNTH_CHARGE_PER_NUTRITION
 		do_sparks(1, FALSE, target_apc)
 
+	if(target_apc.main_status == APC_HAS_POWER)
+		target_apc.charging = APC_CHARGING
```

Comment:
```suggestion
		target_apc.charging = APC_CHARGING
		target_apc.update_appearance()
```

I would also call update_appearance here so that when the synth unplugs it updates the overlay to provide feedback that the APC is in-fact charging